packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -2907,3 +3010,91 @@ test.serial('updateGranuleStatusToFailed() throws error if record does not exist
     }
   );
 });
+
+test.serial('writeGranuleFromApi() saves granule record with publish set to null with publish value set to false to all datastores', async (t) => {
```

Comment:
```suggestion
test.serial('writeGranuleFromApi() saves granule record and sets `published` to false if null value is set', async (t) => {
```
This is a nit but I had a hard time understanding the test statement 😅